tests/testthat/test-run_filter.R

```diff
@@ -0,0 +1,33 @@
+patrick::with_parameters_test_that(
+  "test run_filter",
+  {
+    if(ci_skip == TRUE) skip_on_ci()
+
+    testdata <- file.path("..", "testdata")
+    input_path <- file.path(testdata, "input", filename)
```

Comment:
There are only `mzml` files in `input` directory. Maybe you meant `filtered`?

Reply:
The test file has been updated to the folder containing the input file (~/filtered/run_filter/). [9839836]